src/extension/tab/backend.js

```diff
@@ -0,0 +1,10 @@
+import relay from './tabRelay';
+import { initializeHook } from "./hook";
+import { version as devToolsVersion } from "../manifest.json";
+
+if (document instanceof HTMLDocument) {
+  const script = document.createElement("script");
+  script.textContent = `;(${initializeHook.toString()})(window, "${devToolsVersion}")`;
```

Comment:
Any reason we can't `import { initializeHook } from './hook'` and call it like the function that it is? As long as it gets the right `window`, I think it should work?